src/components/Tag/_tag.scss

```diff
@@ -78,46 +76,33 @@
 
 .#{$prefix}--tag--item {
   position: relative;
+  display: flex;
+  align-items: center;
 }
 
 .#{$prefix}--tag--left {
   cursor: pointer;
-  height: rem(24px);
-  width: rem(24px);
-  position: absolute;
-  top: rem(7px);
   border: 1px solid transparent;
   border-radius: 50%;
-  transition: all 0.35s ease-in;
+  transition: all $transition--expansion ease-in;
+  @include bidi-margin-right(rem(6px));
 
   &:focus {
     outline: none;
     box-shadow: 0 0 0 1px $brand-01, 0 0 0 2px $inverse-01;
   }
-
-  + span {
-    position: relative;
-    @include bidi-left(rem(32px));
-  }
 }
 
 .#{$prefix}--tag--left__success {
-  fill: $brand-01;
-}
-
-.#{$prefix}--tag--left__st {
-  stroke: $text-01;
-  stroke-width: 1;
+  .#{$prefix}--action-icon_img {
```

Comment:
I know, a bit of a BEM anti-pattern here!

Reply:
I'll allow it

Reply:
Thanks A!